webrtc.html

```diff
@@ -3314,7 +3314,7 @@ <h2>Attributes</h2>
               </li>
               <li>
                 <p>If <var>connection</var> has any stopped "sendrecv" or
-                "recvonly" transceivers of type <var>mediaType</var>, jump to
+                "recvonly" transceivers of kind <var>mediaType</var>, jump to
```

Comment:
Maybe s/`mediaType`/`mediaKind` too? We might want to be even more specific. How about:

 "recvonly" transceivers whose `receiver.kind` is *mediaKind*, jump to

?

Reply:
whose *transceiver kind* (with link to the definition)?

I agree that mediaKind would be more consistent language, will update.

Reply:
just kind instead of mediaKind is even more consistent :-)

Reply:
Thanks, I forgot we had the [transceiver kind](http://w3c.github.io/webrtc-pc/#dfn-transceiver-kind) definition.

Reply:
So did I, obviously :)